setup.cfg

```diff
@@ -10,3 +10,14 @@ skip_glob=
     */migrations/*
 profile = black
 known_first_party = apps
+
+[coverage:run]
+source = .
+omit =
+   manage.py
+   *venv/*
```

Comment:
Why does this have a `/` and other dirs don't? I guess we can remove `/`.

Can we add `*env*` as our own instructions tell to create a env by this name?

Can we add `*test*` as tests should not be in coverage.

Reply:
Yup, seems fair to me. I have pushed a commit for this.